src/search.rs

```diff
@@ -13,14 +13,13 @@ use std::{
     time::{Duration, Instant},
 };
 
-use anyhow::{bail, Result};
 use chessie::{Game, Move, MoveList, PieceKind, Position, ZobristKey};
 use uci_parser::{UciInfo, UciResponse, UciSearchOptions};
 
 use crate::{value_of, Evaluator, Score, TTable, TTableEntry};
 
 /// Maximum depth that can be searched
-pub const MAX_DEPTH: u8 = u8::MAX;
+pub const MAX_DEPTH: u8 = u8::MAX / 2;
```

Comment:
Should probably have noted this in the PR, but this change doesn't actually affect anything functionally. A max depth of 127 is fine, as we're never likely to search that deep anyway.